torch/csrc/jit/script/lexer.h

```diff
@@ -83,7 +83,9 @@ namespace script {
   _(TK_FOR, "for", "for")                        \
   _(TK_IN, "in", "in")                           \
   _(TK_STARRED, "starred", "")                   \
-  _(TK_UNARY_MINUS, "unary minus", "")
+  _(TK_UNARY_MINUS, "unary minus", "")           \
+  _(TK_POW, "pow operator", "**")                \
+  _(TK_MATMUL, "matmul operator", "@")
```

Comment:
For single-character tokens like `@` we just use the token itself rather than defining a `TK_*` symbol. Can we remove TK_MATMUL to follow this pattern?